code/drasil-example/swhs/lib/Drasil/SWHS/Unitals.hs

```diff
@@ -415,7 +415,7 @@ outputs = [tempW, tempPCM, watE, pcmE]
 tempW = cuc' "tempW"
   (nounPhraseSP "temperature of the water")
   "the average kinetic energy of the particles within the water" 
-  (sub (eqSymb temp) lWater) centigrade (Vect Real)
+  (sub (eqSymb temp) lWater) centigrade (vectND "n" Real) -- TODO: investigate the implications of this
```

Comment:
I spoke to Dr. @smiths today about the SWHS example. We think that the type of the [tempW constraint](https://github.com/JacquesCarette/Drasil/blob/905a96bb60f83e627241c0a5fbc419c7fe87b146/code/drasil-example/swhs/lib/Drasil/SWHS/Unitals.hs#L415) is wrong. It should probably be `Real` if the variable is seen as a single Celsius temperature (which it how it's defined earlier on). But it gets a bit confusing because some parts of the document refers to it as T_w(t), meaning the temperature over time, which implies that it is a function. In code this function may be implemented as a discretized vector of temperatures at different time intervals, which may be where the confusion came from.

Reply:
Wonderful! I was sure that work like this was going to find more bugs.